client/src/common/components/ListViewItem.jsx

```diff
@@ -71,6 +71,7 @@ class ListViewItem extends PureComponent {
             })}
             disabled={pending}
             onClick={this.handleFavClick}
+            title="Add to favourites"
```

Comment:
maybe change to sth like this title={`$ {isFavourite ? 'Add to' : 'Remove from`} favourites`}